gradle-jdks/src/main/java/com/palantir/gradle/jdks/ToolchainsPlugin.java

```diff
@@ -89,7 +118,11 @@ public void apply(Project rootProject) {
             task.getDaemonJavaVersion().set(jdksExtension.getDaemonTarget());
             task.getJavaVersionToJdkDistros()
                     .putAll(rootProject.provider(() -> JdkDistributionConfigurator.getJavaVersionToJdkDistros(
-                            rootProject, jdkDistributions, jdksExtension)));
+                            rootProject,
+                            jdkDistributions,
+                            jdksExtension,
+                            task.getIncludeAllJdks().get(),
+                            task.getIncludeJavaMajorVersions().get())));
```

Comment:
do you ever expect these to carry task dependencies?

if so, then probably want to do a zip instead

Reply:
oh, this is on the task itself, it's a bit confusing to grok this